spec/cspec/c/kernel.mk

```diff
@@ -109,8 +105,8 @@ ${KERNEL_BUILD_ROOT}/.cmake_done: ${KERNEL_DEPS} ${CONFIG_DOMAIN_SCHEDULE}
 		-DCROSS_COMPILER_PREFIX=${TOOLPREFIX} \
 		-DCMAKE_TOOLCHAIN_FILE=${SOURCE_ROOT}/gcc.cmake \
 		-DKernelDomainSchedule=${CONFIG_DOMAIN_SCHEDULE} \
-		-DUMM_TYPES=$(abspath ${UMM_TYPES}) \
-		-DCSPEC_DIR=${CSPEC_DIR} ${KERNEL_CMAKE_EXTRA_OPTIONS} \
+		-DUMM_TYPES=$(abspath ${UMM_TYPES}) -DCSPEC_DIR=${CSPEC_DIR} \
+		${KERNEL_CMAKE_OPTIMISATION} ${KERNEL_CMAKE_EXTRA_OPTIONS} \
```

Comment:
Just saw that we should make the same change to the config target. Some of the other options depend on optimisation options and we want the config build to be in sync with the main build. (The proofs will fail if they are not, so it doesn't need to be trusted, but of course we don't want them to fail).

Reply:
Right, I had assumed it wouldn't matter, but you're right that we should ensure consistency. I will update.